libs/verify-android-ci-environment.sh

```diff
@@ -53,8 +53,8 @@ location of your Java installation."
 fi
 
 JAVA_VERSION=$("$JAVACMD" -version 2>&1 | grep -i version | cut -d'"' -f2 | cut -d'.' -f1-2)
-if [[ "${JAVA_VERSION}" != "1.8" ]]; then
-  echo "Incompatible java version: ${JAVA_VERSION}. JDK 8 must be installed."
+if [[ "${JAVA_VERSION}" != "1.11" ]]; then
+  echo "Incompatible java version: ${JAVA_VERSION}. JDK 11 must be installed."
   echo "Try switching versions and re-running. Using sdkman: sdk install java 8.0.282+8.hs-adpt || sdk use java 8.0.282+8.hs-adpt"
```

Comment:
This line should be updated as well, but I don't use `sdkman` and I'm a bit afraid of installing it and having 1 more thing that installs JDKs to me system.  Does anyone else have a suggestion for this?

Reply:
`sdk install java 11.0.3.hs-adpt || sdk use 11.0.3.hs-adpt`

Reply:
sdk install java 11.0.3.hs-adpt || sdk use 11.0.3.hs-adpt